location/models.py

```diff
@@ -22,5 +22,5 @@ class Location(models.Model):
 
     def __str__(self):
         if self.name is None:
-            return self.id
+            return str(self.id)
```

Comment:
Perhaps it's a Python version error, but when I tried to run `link_locations` locally it complained that it couldn't format a string with an integer when printing this ID to the console.